qiskit_addon_sqd/fermion.py

```diff
@@ -231,16 +231,17 @@ def optimize_orbitals(
         )
 
         # Generate the one and two-body reduced density matrices from latest wavefunction amplitudes
-        dm1, dm2_chem = myci.make_rdm12(amplitudes, num_orbitals, (num_up, num_up))
+        dm1, dm2_chem = myci.make_rdm12(amplitudes, num_orbitals, (num_up, num_dn))
         dm2 = np.asarray(dm2_chem.transpose(0, 2, 3, 1), order="C")
+        dm11, dm22 = myci.make_rdm1s(amplitudes, num_orbitals, (num_up, num_dn))
```

Comment:
The variable names don't seem very descriptive to me. I suggest `dm1a, dm1b` instead.